src/Symfony/Bundle/FrameworkBundle/Resources/config/fragment_renderer.xml

```diff
@@ -11,7 +11,7 @@
 
     <services>
         <service id="fragment.handler" class="Symfony\Component\HttpKernel\DependencyInjection\LazyLoadingFragmentHandler">
-            <argument type="service" id="service_container" />
+            <argument /> <!-- fragment renderer locator -->
```

Comment:
that's because of https://github.com/symfony/symfony/blob/master/src/Symfony/Component/EventDispatcher/EventDispatcher.php#L127 which requires an instance
but this PR now has a fix you will like :)